examples/pybids_tutorial.ipynb

```diff
@@ -1855,7 +1855,7 @@
     }
    ],
    "source": [
-    "# Can check if file a dataset top\n",
+    "# Can check if a filepath represents a file part of the dataset (i.e. the  top\n",
```

Comment:
There's a missing close-paren (`)`), but also, this just feels grammatically weird. What about:

```suggestion
    "# Can check if a file path is at the top level of the dataset\n",
```

Reply:
Thanks for the review @effigies !

I failed to build a proper sentence/did not finish and commit properly while switching between branches.

5b49aba rewords it following your advice.